diff --git a/dist/support/helpers.js b/dist/support/helpers.js
index 4346548126fcf7e3687f243ee53fd0456778c29c..73cefe121e927285d305d57e89f3c87f8d2fe289 100644
--- a/dist/support/helpers.js
+++ b/dist/support/helpers.js
@@ -42,7 +42,12 @@ function getReflectionHeadingLevel(reflection, groupByReflections) {
 exports.getReflectionHeadingLevel = getReflectionHeadingLevel;
 function getReflectionTitle(reflection, fullname = false, typeParams = false) {
     var _a, _b;
-    const md = [
+    const md = reflection.kindOf([
+        typedoc_1.ReflectionKind.Module,
+        typedoc_1.ReflectionKind.Namespace,
+    ]) ? [
+        "`langchain/" + (fullname ? reflection.getFullName() : reflection.name) + "`",
+    ] : [
         (0, utils_1.escapeChars)(fullname ? reflection.getFullName() : reflection.name),
     ];
     if ((_a = reflection.signatures) === null || _a === void 0 ? void 0 : _a.length) {
diff --git a/dist/theme.js b/dist/theme.js
index f6eaa7c35c0b377a019f4f948531917eef5fe302..dd9e5ac90b4f86740fca3d0f0b55369efcfcded6 100644
--- a/dist/theme.js
+++ b/dist/theme.js
@@ -131,11 +131,6 @@ class MarkdownTheme extends typedoc_1.Theme {
                     parts.splice(namespaceIndex, 0, this.mappings[typedoc_1.ReflectionKind.Namespace].directory);
                 }
             });
-            if (reflection.kindOf(typedoc_1.ReflectionKind.Namespace))
-                parts[parts.length - 1] = `${typedoc_1.ReflectionKind.singularString(typedoc_1.ReflectionKind.Namespace).toLowerCase()}.${parts[parts.length - 1]}`;
-        }
-        if (!isModuleOrNamespace) {
-            parts[parts.length - 1] = `${typedoc_1.ReflectionKind.singularString(reflection.kind).toLowerCase()}.${parts[parts.length - 1]}`;
         }
         return parts.join(this.flattenOutputFiles ? '.' : '/') + '.md';
     }
